src/systemctl/systemctl.c

```diff
@@ -3348,8 +3348,10 @@ static int logind_check_inhibitors(enum action a) {
                 if (!sv)
                         return log_oom();
 
-                if ((pid_t) pid < 0)
-                        return log_error_errno(ERANGE, "Bad PID %"PRIu32": %m", pid);
+                if (!pid_is_valid((pid_t) pid)) {
+                        log_error("Bad PID %"PRIu32": %m", pid);
```

Comment:
`pid_is_valid()` doesn't set `errno`.  Previous version was correct, I think.

Reply:
ouch, indeed

Reply:
(well, it shows ERANGE, pulling this error out of thin air... we shouldn't do that really, there's no ERANGE received here anywhere, hence we shouldn't confuse the user seeing this with it)